posthog/api/test/test_dashboard.py

```diff
@@ -56,12 +54,7 @@ def test_update_dashboard(self):
         )
         response = self.client.patch(
             f"/api/projects/{self.team.id}/dashboards/{dashboard.id}",
-            {
-                "name": "dashboard new name",
-                "creation_mode": "duplicate",
-                "tags": ["official", "engineering"],
```

Comment:
Hmm I'm not sure I'm following. 

These are being removed from the non-ee tests because tagging is part of taxonomy which is a licensed feature which doesn't belong in the standard `posthog/api/test` folder. It's not a permissions thing but a licensing difference. The tests here (https://github.com/PostHog/posthog/pull/8529/files#diff-6d57b94f5f2d3d28012fb5c8e8aca2e4b732ade5117e40b664dd097fc42bb75eR72-R76) require licenses so they're placed in `ee/api/test`. Am I understanding correctly?

Reply:
I think you're correct and I'm wrong. Thanks for clarifying!